src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -18,6 +18,11 @@ export default function ProgressBar({ percent, label }) {
 }
 
 ProgressBar.propTypes = {
+  /**
+   * The `aria-label` to use for the component
+   */
+  label: PropTypes.isRequired,
```

Comment:
According to [an investigation](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/357#issuecomment-765665591), there are only two uses of `<ProgressBar>` in `vets-website`, neither of which use that `label` prop. So my questions are:

- _Should_ this be required?
- Should this be a prop at all?
    - If not, should the hard-coded value be something like `${percent} percent finished`?